src/DatadogLoggingService.js

```diff
@@ -28,13 +28,14 @@ class DatadogLoggingService extends NewRelicLoggingService {
   }
 
   initialize() {
+    const datadogVersion = process.env.DATADOG_VERSION || process.env.APP_VERSION || '1.0.0';
     datadogRum.init({
-      applicationId: process.env.DATADOG_APPLICATION_ID,
-      clientToken: process.env.DATADOG_CLIENT_TOKEN,
-      site: process.env.DATADOG_SITE,
-      service: process.env.DATADOG_SERVICE,
-      env: process.env.DATADOG_ENV,
-      version: process.env.DATADOG_VERSION,
+      applicationId: process.env.DATADOG_APPLICATION_ID || '',
```

Comment:
[curious] What happens when `datadogRum.init` and `datadogLogs.init` are called with these values as an empty string? Are some of these values required by DD and others optional, e.g., would it throw an error if `DATADOG_APPLICATION_ID` is not supplied? 

[suggestion] I wonder if it might make sense to only call `datadogRum.init` and `datadogLogs.init` when the required config values are present to avoid passing empty strings. Perhaps something like:

```jsx
initialize() {
  const requiredDatadogConfig = [
    process.env.DATADOG_APPLICATION_ID,
    process.env.DATADOG_CLIENT_TOKEN,
    process.env.DATADOG_SITE,
  ];
  const hasRequiredDatadogConfig = requiredDatadogConfig.every(value => !!value);

  // Do not attempt to initialize Datadog if required config settings are not supplied.
  if (!hasRequiredDatadogConfig) { return; }

  // Otherwise, initialize Datadog.
  datadogRum.init({ ... });
  datadogLogs.init({ ... });
}
```

Reply:
Datadog doesn't crash if required config isn't present. It just log console error. But yes, i'll add this additional check
<img width="1673" alt="image" src="https://github.com/edx/frontend-logging/assets/42172960/66fdcd10-7407-4ede-95fc-ed58eebd1773">

Reply:
Added check on `process.env.DATADOG_APPLICATION_ID` and `process.env.DATADOG_CLIENT_TOKEN` only since these two are main fields